test/conformance/testdata/ineligible_endpoints.yaml

```diff
@@ -295,3 +295,9 @@
 - endpoint: replaceNetworkingV1ServiceCIDRStatus
   reason: Kubernetes distribution would reasonably not allow this action via the API
   link: https://github.com/kubernetes/enhancements/pull/4983#issuecomment-2541699445
+- endpoint: connectCoreV1PostNamespacedPodExec
+  reason:  Non-essential debug endpoints, opting to exclude them
+  link: https://github.com/kubernetes/kubernetes/issues/112778
+- endpoint: connectCoreV1PostNamespacedPodPortforward
+  reason:  Non-essential debug endpoints, opting to exclude them
+  link: https://github.com/kubernetes/kubernetes/issues/112778
```

Comment:
I am not sure whether there is consensus that those endpoints aren't required for conformance.

My takeaway from previous discussions was that "kubectl exec" and "kubectl portfoward" are so essential for various tutorials that it would be surprising if they didn't work in a conformant cluster. Is that correct?

Reply:
So to summarize: other operations for exec/portforward are part of conformance, just not the ones listed here. Adding them to `ineligible_endpoints.yaml` makes sense to me, but let's come up with a better explanation.